src/main/java/ita/softserve/course_evaluation_analytics/repository/impl/AnalystRepositoryImpl.java

```diff
@@ -0,0 +1,68 @@
+package ita.softserve.course_evaluation_analytics.repository.impl;
+
+import ita.softserve.course_evaluation_analytics.entity.Course;
+import ita.softserve.course_evaluation_analytics.entity.FeedbackRequest;
+import ita.softserve.course_evaluation_analytics.mapper.CoursesMapper;
+import ita.softserve.course_evaluation_analytics.mapper.FeedbackRequestMapper;
+import ita.softserve.course_evaluation_analytics.repository.AnalystRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import javax.sql.DataSource;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+@Repository
+public class AnalystRepositoryImpl implements AnalystRepository {
+
+    private NamedParameterJdbcTemplate namedParameterJdbcTemplate;
+    private JdbcTemplate jdbcTemplate;
```

Comment:
use only ```NamedParameterJdbcTemplate```

Reply:
some methods for one object is not present in NameParameterJdbcTemplate

Reply:
```namedParameterJdbcTemplate.query()``` is what you are looking for